exercises/concept/elons-toy-car/.docs/introduction.md

```diff
@@ -2,7 +2,7 @@
 
 ## Classes
 
-The primary object-oriented construct in Java is the _class_, which is a combination of data (_fields_) and behavior (_methods_).
+The primary object-oriented construct in Java is the _class_, which is a combination of data (_fields_) and behaviour (_methods_).
```

Comment:
"behavior" is the [American spelling](https://grammarist.com/spelling/behavior-behaviour/). Exercism uses American spelling for the content (see the [Exercism style guide](https://exercism.org/docs/building/markdown/style-guide)).

Reply:
Okay, I will change it.